template/src/ducks/counter.js

```diff
@@ -0,0 +1,27 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const slice = createSlice({
+  name: "counter",
+  initialState: {
+    value: 0
+  },
+  reducers: {
+    increment: state => {
+      state.value += 1;
+    },
+    decrement: state => {
+      state.value -= 1;
+    }
+  }
+});
+
+const selectCount = state => state.counter.value;
+
+const selectors = {
+  selectCount
+};
+
+const { actions } = slice;
+
+export { actions, selectors, slice };
```

Comment:
Personally I find separate `export` lines easier to read.

Reply:
you could argue that beginners already know you can do separate exports.
 this would give them a chance to try something new. 
"oh, I didn't know you could do that"  kinda thing.

Reply:
I would prefer simplicity and convention in a template.

Reply:
Done. Should I be exporting an object of selectors or individually exporting each selector? Same for actions?